app/views/pages/cookies.html.erb

```diff
@@ -0,0 +1,43 @@
+<h1>Cookies</h1>
```

Comment:
Can you wrap the page in `<div class="text-page">` like the privacy page so that it picks up the correct styles